### ISSUE TITLE
Whitespace glitch when updating

### ISSUE THREAD

**Issue by papandreou** [mafintosh/diffy#2]
Thanks for creating this library -- it looks really promising.

I ran into a problem while trying to render a live-updated table. It can be boiled down to:

```js
const frames = [
`
┌─┐
│ │
`,
`
┌────┐
│    │
`
];

const diffy = require('diffy')();
const trim = require('diffy/trim');
let nextFrameNumber = 0;

diffy.render(() => trim(`
    ${frames[nextFrameNumber++]}
`));

setTimeout(() => diffy.render(), 1000);
```

## Expected output:

```
┌─┐
│ │
```

and then:

```
┌────┐
│    │
```

## Actual output:

```
┌─┐
│ │
```

and then:

```
┌────┐
││
```

Reproduced with the Gnome Terminal on Ubuntu.

Seems like removing the whitespace between `` trim(` `` and `${` corrects the output, so I hope it's an easy fix :)

**Comment by mafintosh** [comment]
Can reproduce. Will get it fixed, thanks!

**Comment by mafintosh** [comment]
Fixed in 1.3.1. Was a bug in the trim method.
